example/server.js

```diff
@@ -15,7 +15,20 @@ app.get('/', function (res, res, next) {
           firstName: 'Bapu'
       }]
   })
-})
+});
+
+app.get('/error', (req, res, next) => {
+    next(new Error("oops something went wrong"));    
+});
+
+app.use((err, req, res, next) => {
```

Comment:
Do you need to define this? I thought the default error handler would do this for us.